apps/api/src/persistence/database/get-arango-collection-ids.ts

```diff
@@ -3,23 +3,19 @@ import { ArangoCollectionID } from './types/ArangoCollectionId';
 
 // TODO [design] Should we tie this to the model constructors?
 const arangoCollectionIDs: {
-  [k in EntityType]: ArangoCollectionID;
+    [k in EntityType]: ArangoCollectionID;
 } = {
-  term: 'terms',
-  vocabularyList: 'vocabulary_lists',
-  tag: 'tags',
+    term: 'terms',
```

Comment:
Isn't this another source of truth for the entities?

Reply:
This is not a source of truth for the entities. See the type:
```ts
    [k in EntityType]: ArangoCollectionID;
```
Clearly `EntityType` (derived from the type of `entityTypes`) is the source of truth here.